packages/libllvm/clang.subpackage.sh

```diff
@@ -5,11 +5,17 @@ bin/*clang*
 bin/*cpp
 bin/*g++
 bin/*gcc
-bin/scan-build
+bin/git-clang-format
+bin/scan-*
+include/clang*
+include/omp.h
 lib/clang
+lib/cmake/clang
+lib/libclang*so
```

Comment:
The `scan-build` and `scan-view` binaries are normally packaged with clang and `git-clang-format` could be useful. The `omp.h` header should be included here, since the library is too.

The `libclang*so` shared libraries were previously included with the `libllvm` package, and there are still 34 `libclang*a` static libraries included with `libllvm-static`. I can keep them there, include them here, or create a separate `libclang-static`, whatever is wanted.